frontend/src/data/tasks.json

```diff
@@ -1142,8 +1124,8 @@
       },
       {
         "answerKey": "yes-income",
-        "descriptionEn": "If you continue working, this may affect the amount of tax you need to pay on your income.\n\nIf you can afford to delay collecting your public pensions, the longer you wait, the higher your monthly amounts will be for life.\n\nIf you continue to work while collecting your CPP pension, and are under age 70, you can continue to take part in the CPP. Your CPP contributions will go toward post-retirement benefits (PRB), which will increase your monthly pension amount.",
-        "descriptionFr": "Si vous continuez à travailler, cela peut avoir une incidence sur le montant de l'impôt que vous devez payer sur vos revenus.\n\nSi vous pouvez vous le permettre, il est préférable de retarder le paiement de votre pension; plus vous attendez, plus vos montants mensuels seront élevés jusqu'à la fin de votre vie.\n\nSi vous continuez à travailler tout en recevant votre pension du RPC et que vous avez moins de 70 ans, vous pouvez continuer à cotiser au RPC. Vos cotisations seront applicables aux prestations après-retraite (PAR), ce qui augmentera votre revenu de retraite.",
+        "descriptionEn": "If you continue working, this may affect the amount of tax you need to pay on your income.\n\nIf you can afford to delay collecting your public pensions, the longer you wait, the higher your monthly amounts will be for life.\n\nIf you continue to work while collecting your CPP retirement pension, and are under age 70, you can continue to take part in the CPP. Your CPP contributions will go toward post-retirement benefits (PRB), which will increase your monthly pension amount.",
+        "descriptionFr": "Si vous continuez à travailler, cela peut avoir une incidence sur le montant de l'impôt que vous devez payer sur vos revenus.\n\nSi vous pouvez vous le permettre, il est préférable de retarder le paiement de votre pension; plus vous attendez, plus vos montants mensuels seront élevés jusqu'à la fin de votre vie.\n\nSi vous continuez à travailler tout en recevant votre pension de retraite du RPC et que vous avez moins de 70\u00a0ans, vous pouvez continuer à cotiser au RPC. Vos cotisations seront applicables aux prestations après-retraite (PAR), ce qui augmentera votre revenu de retraite.",
```

Comment:
Non-breaking space `\u00a0` is not needed with `70 ans`.

Reply:
Removed `\u00a0`